src/Screens/Onboarding/DisplayName.tsx

```diff
@@ -35,6 +35,9 @@ const DisplayName = ({ navigation }: Props) => {
     });
   };
 
+  const isValidDisplayName =
+    displayName.length >= 2 && displayName.length <= 100;
```

Comment:
The validation logic could be in its own function and that could be unit tested, but these are very simple rules.

Reply:
Yes, it is better to do like that. I'll make a commit in the following days :p